app/presenters/summary/html_sections/base_children_details.rb

```diff
@@ -0,0 +1,65 @@
+module Summary
+  module HtmlSections
+    class BaseChildrenDetails < Sections::BaseSectionPresenter
+      protected
+
+      # rubocop:disable Metrics/MethodLength
+      def personal_details(child)
+        [
+          FreeTextAnswer.new(
+            :child_full_name,
+            child.full_name,
+            change_path: edit_steps_children_names_path(child)
+          ),
+          DateAnswer.new(
+            :child_dob,
+            child.dob,
+            change_path: edit_child_details_path(child, 'dob_dd')
+          ),
+          FreeTextAnswer.new(
+            :child_age_estimate,
+            child.age_estimate,
+            change_path: edit_child_details_path(child, 'age_estimate')
+          ),
+          Answer.new(
+            :child_sex,
+            child.gender,
+            change_path: edit_child_details_path(child, 'gender_female')
+          ),
+        ]
+      end
+      # rubocop:enable Metrics/MethodLength
+
+      def relationships(child)
```

Comment:
I believe the relationships will be best placed in each of the applicant/respondent sections, when we have them, but for now it is ok to leave them here. We can move them around later on.